src/links/link-generator.ts

```diff
@@ -453,7 +454,13 @@ function getEntryPointForAngularComponent(
   const componentRoot: string = component.writtenPath || componentMap.rootDir;
   if (componentMap.origin === COMPONENT_ORIGINS.AUTHORED) return null;
   const content = getLinkToFileContent(component.mainFile, []);
-  const filePath = path.join(componentRoot, ANGULAR_BIT_ENTRY_POINT_FILE);
+  const filePath = ANGULAR_BIT_ENTRY_POINT_FILE.find((entryPoint) => {
+    const fp = path.join(componentRoot, entryPoint);
+    return existsSync(fp);
```

Comment:
@davidfirst can you confirm that this will work? I'm not sure if componentRoot is absolute or relative, but I guess it's a valid path either way

Reply:
I added `consumer.toAbsolutePath`

Reply:
Looks good.